doc/how_to/custom_components/reactive_esm/reactive_esm_layout.md

```diff
@@ -0,0 +1,218 @@
+# Create Layouts With ESM Components
+
+In this guide we will show you how to build custom layouts `JSComponent`, `ReactComponent` or `PreactComponent`.
+
+## Layout a single Panel Component
+
+You can layout a single Panel component as follows.
+
+:::::{tab-set}
+
+::::{tab-item} `JSComponent`
+
+```{pyodide}
+import panel as pn
+from panel.custom import JSComponent, Child
+
+pn.extension()
+
+class LayoutSingleObject(JSComponent):
+    object = Child()
```

Comment:
For widgets it unfortunately shows the widget value and not the widget.

![image](https://github.com/holoviz/panel/assets/42288570/4405654c-4227-44fb-bd4e-73550b444d09)


(Same issue for ReactComponent)

Reply:
Will fix, have to disable automatic `allow_refs=True` for child components.

Reply:
Fixed.